### DISCUSSION TITLE
oxalis-outbound component

### DISCUSSION THREAD

**Discussion by DeDa22** [OxalisCommunity/Oxalis-AS4#214] [Q&A]
Hello
I have few questions
1) Is it oxalis-standalone good enough to be used in production to send documents?
2) Do you have any link or documentation about how to send documents via oxalis-outbound, any good practice or maybe do you have kinda a black box server which can be used  for sendind documents.
3) How to examine status of sending document (to be sure that customer really receive document which we did send.)

FYI we are not Java shop, we are .NET shop and we are interesting like to use oxalis for receiving and sending documents as Black Box from our .NET applications/servers is it any chance to do it ?? So far we already did pass test and receive prod cert.

with regards
Boris

**Comment by dladlk** [comment]
Hello
1. Oxalis-standalone is just an illustrative program showing how to send documents with Oxalis, I guess everyone integrate it into some more adhoc solution
2. You can try to look into Nemhandel Edelivery extension to Oxalis - it has both sending/receiving possibilities, but adds additional ASIC containers validations, and should give you an inspiration - https://rep.erst.dk/git/openebusiness/nemhandeledelivery/oxalis-as4 (guidelines how to install and run are in Danish here: https://rep.erst.dk/git/openebusiness/common/-/blob/master/guidelines/vejledning_-_drift.pdf) - but you need to replace NemhandelPersister class with something your own and it requires java skills.
3. If it did not throw exception - the document is sent, as a receiver returns back signed receipt. In some implementations, positive receipt is sent back additionally, but it is not mandatory in Peppol general spec.

**Comment by DeDa22** [comment]
@dladlk 
Thank You very much
